### DISCUSSION TITLE
Events 'n Metas: Play a configurable sound on reminder

### DISCUSSION THREAD

**Discussion by sLy1337** [blish-hud/Community-Module-Pack#160] [Module Suggestions]
Hi,
I have recently downloaded the Events 'n Metas module and I like it very much.
In addition to the time when the reminder pops up which should be configurable (I saw that this will be included in a future version) I would suggest to include a notification sound.
My suggestion would be a  configurable "standard" notification sound which is used if no other is selected and an individual sound for each event which can be configured optionally. If configured, it is used instead of the standard sound. 

Thank you in advance and keep up the good work!

Best regards!

**Comment by foxon81** [comment]
please make key combo.or add option to klick key where call this addon to watch what events is coming...
